Traceback (most recent call last):
  File "/home/cuongnq1/graph-neural-pde/src/run_grand_ex.py", line 427, in <module>
    main(opt)
  File "/home/cuongnq1/graph-neural-pde/src/run_grand_ex.py", line 210, in main
    model = GrandExtendDiscritizedNet(opt, dataset, device).to(device)
  File "/home/cuongnq1/graph-neural-pde/src/grand_discritized.py", line 105, in __init__
    super().__init__(opt["hidden_dim"], opt, data, device)
  File "/home/cuongnq1/graph-neural-pde/src/grand_discritized.py", line 75, in __init__
    self.step_size = torch.Tensor([opt["step_size"]]).to(device)
RuntimeError: CUDA error: out of memory
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.